x/gamm/keeper/migrate.go

```diff
@@ -206,3 +194,20 @@ func (k Keeper) UpdateMigrationRecords(ctx sdk.Context, records []types.Balancer
 	})
 	return nil
 }
+
+// CheckIfPoolMigrationLinkIsAuthorized checks if a governance sanctioned link exists between the provided balancer and concentrated pool.
+func (k Keeper) GetLinkedConcentratedPool(ctx sdk.Context, poolIdLeaving uint64) (poolIdEntering uint64, err error) {
```

Comment:
Spec doesn't apply to method

Reply:
https://github.com/osmosis-labs/osmosis/pull/4129/commits/4f62673f13b377e76a5feecccd511c09e4c08f9c